### PR TITLE
New Resource: `azurerm_sentinel_log_analytics_workspace_onboarding`

### PR DESCRIPTION
**Pull request by ziyeqf**
Test
---
```
❯ TF_ACC=1 go test -v ./internal/services/sentinel -run=TestAccSecurityInsightsSentinelOnboardingState -timeout=600m
=== RUN   TestAccSecurityInsightsSentinelOnboardingState_basic
=== PAUSE TestAccSecurityInsightsSentinelOnboardingState_basic
=== RUN   TestAccSecurityInsightsSentinelOnboardingState_ToggleCmkEnabled
=== PAUSE TestAccSecurityInsightsSentinelOnboardingState_ToggleCmkEnabled
=== RUN   TestAccSecurityInsightsSentinelOnboardingState_requiresImport
=== PAUSE TestAccSecurityInsightsSentinelOnboardingState_requiresImport
=== CONT  TestAccSecurityInsightsSentinelOnboardingState_basic
=== CONT  TestAccSecurityInsightsSentinelOnboardingState_requiresImport
=== CONT  TestAccSecurityInsightsSentinelOnboardingState_ToggleCmkEnabled
--- PASS: TestAccSecurityInsightsSentinelOnboardingState_basic (294.44s)
--- PASS: TestAccSecurityInsightsSentinelOnboardingState_requiresImport (361.96s)
--- PASS: TestAccSecurityInsightsSentinelOnboardingState_ToggleCmkEnabled (5564.88s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/sentinel      5564.906s
```

### CODE REVIEW

File: website/docs/r/sentinel_log_analystic_workspace_onboard.html.markdown [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_log_analytics_workspace_onboard"
```
Comment (katbyte) [inline website/docs/r/sentinel_log_analystic_workspace_onboard.html.markdown:4]: given the description below uses onboarding
```suggestion
page_title: "Azure Resource Manager: azurerm_sentinel_log_analytics_workspace_onboarding"
```

as "onboard" doesn't seem like the right name?

File: website/docs/r/sentinel_log_analystic_workspace_onboard.html.markdown [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-law"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_security_insights_sentinel_onboarding_state" "example" {
```
Comment (katbyte) [inline website/docs/r/sentinel_log_analystic_workspace_onboard.html.markdown:28]: and its onboarding here, thou with state?
```suggestion
resource "azurerm_security_insights_sentinel_onboarding" "example" {
```

**Review by katbyte — CHANGES_REQUESTED** [review]

File: website/docs/r/sentinel_log_analystic_workspace_onboard.html.markdown [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
```
Comment (stephybun) [inline website/docs/r/sentinel_log_analystic_workspace_onboard.html.markdown:1]: There's a typo in this filename `sentinel_log_analystic_workspace_onboard.html.markdown` -> `sentinel_log_analytics_workspace_onboarding.html.markdown`

**Review by katbyte — APPROVED** [review]
LGTM 🍄

**Pull request merged**